block_my_feedback.php

```diff
@@ -120,8 +125,7 @@ public function fetch_feedback() : array {
                         AND gi.hidden < UNIX_TIMESTAMP()
                         AND gg.timemodified >= :since AND gg.timemodified <= UNIX_TIMESTAMP()
                         AND gg.userid = :userid
-                ORDER BY gg.timemodified DESC
-                LIMIT 5";
```

Comment:
have we considered how many items are likely to be returned without this limit 5 ?

I know there is the right logic down the line to limit the displayed items to 5 but what if this returns 5 000 items and takes 5 minutes to load as a result?

Answering my own question: limiting to last 3 months via $since parameter above, so this won't return many records.